tools/SeeDot/README.md

```diff
@@ -40,11 +40,12 @@ optional arguments:
                         procedure (1 for a single-class classification problem)
   -t , --target         Target device ['x86', 'arduino', 'm3']
   -s , --source         model source type ['seedot', 'onnx', 'tf']
+  -l , --log            Logging level (in increasing order) ['ERROR', 'CRITICAL', 'WARNING', 'INFO', 'DEBUG']
```

Comment:
Can we have these options as lowercase please, to keep in line with how we offer other command line options?